Automattic/wp-calypso
# Comments: Add HTML toolbar for the edit textarea

Fix #21663
Depends on #22042

Add an HTML toolbar to help adding quick tags when editing comments in the Comments Management section.

<img width="856" alt="screen shot 2018-02-01 at 12 07 16" src="https://user-images.githubusercontent.com/2070010/35677671-789b3078-0748-11e8-97f8-4a569949c9c4.png">

## Testing instructions

Open `/comments` and click on Edit on some comments.
Use the new HTML toolbar to open and close tags.
Make sure the inserted tags feel "right" (e.g. the expected newlines or indents).

### Note

While testing this, it could happen that clicking on Edit doesn't keep the comment on screen.
It's a known bug, taken care in https://github.com/Automattic/wp-calypso/pull/22125.
Please disregard it here.
